tools/generator/templates/player/c/Makefile.jinja2

```diff
@@ -0,0 +1,44 @@
+# SPDX-License-Identifier: GPL-2.0-or-later
```

Comment:
Almost nothing is done in this file yet, because we need to figure out how to do it properly first

Reply:
Don’t we want to at least add the C sources into the objs variable before merging?